src/components/common/TextContent.test.tsx

```diff
@@ -13,7 +13,7 @@ const markdownText = '# Lorem Ipsum \n\n[Link](https://broadinstitute.org)';
 
 const htmlText = '<a href="www.badsite.com">A bad day</a>';
 
-describe('WithoutStylesMarkdownContent', () => {
+describe('TextContent', () => {
```

Comment:
Why the name change here? Is it sonar related? This seems less descriptive.

Reply:
`WithoutStylesMarkdownContent` is the name of a different component, I thought it was a copy-paste error